### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by ThePortugueseMan**

### CODE REVIEW

**Comment by ThePortugueseMan** [comment]
Codacity is throwing errors related to the console table ext, and one about an empty while statement made to wait for a key press

File: ThePortugueseMan.FlashCards/Screens/StudySessionMenu.cs [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        int stackViewId, stackIndex;
+        StudySession studySession = new();
+        Stack stack = new();
+
+        StacksMenu stacks = new();
+        stacks.DisplayStackList(dbCmd.Return.AllStacks(), "CHOOSE");
+
+        do
+        {
+            stackViewId = askInput.PositiveNumber("Choose the stack you want to study, or 0 to return");
```
Comment (TheCSharpAcademy) [inline ThePortugueseMan.FlashCards/Screens/StudySessionMenu.cs:52]: Hey! Project is looking great! Just a bit unsure what is the behavior expected in the study session area? As an user it wasn't clear what the functionality is. I think adding a bit more explanation will do? 

image

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

File: ThePortugueseMan.FlashCards/Screens/StudySessionMenu.cs (hunk 1 above)
Comment (ThePortugueseMan) [reply ThePortugueseMan.FlashCards/Screens/StudySessionMenu.cs:52]: Hey, thanks for the feedback! I've updated the code.

image


The user was supposed to press the left/right arrow if the got the answer wrong/right. Similar to how a traditional flashcard system would work.

I hope that's clear, and thanks for the review!

**Review by TheCSharpAcademy — APPROVED** [review]
Hey man! Going to approve now. Well done!

For future project I recommend you add libraries as nuget packages though, when possible, so you're not mixing your code with the library. 😁

Also, since now you're moving up a belt, coding standards will be strictly enforced. The Academy gets more demanding on our best students so they keep getting better and better. 😃

**Pull request merged**